### ISSUE TITLE
NVDA could not start securely.

### ISSUE THREAD

**Issue by davidacm** [nvaccess/nvda#14063]
### Steps to reproduce:

1. Download beta 3 of NVDA, the current alphas, or the version 2022.2.1.
2. Open the installer.

### Actual behavior:
NVDA launch the following message:
"NVDA could not start securely."
"NVDA failed to register session tracking. While this instance of NVDA is running, your desktop will not be secure when Windows is locked. Restart NVDA?"

If NVDA is restarted, the same error appears.

This error appears even if NVDA is executed with the option "run as administrator".

The error disappears if you restart the system. But if you reinstall another version of NVDA, for example change between the latest stable 2022.2.1 and the 2022.3beta3, the error appears again until the system is restarted.

### Expected behavior:
NVDA runs normally, and don't show the message.

### NVDA logs, crash dumps and other attachments:

ERROR - winAPI.sessionTracking.register (00:00:32.707) - MainThread (32392):
Unexpected error registering session tracking.
PermissionError: [WinError 5] Access is denied.

#### NVDA installed/portable/running from source:

Installer, portable, and installed environments.

#### NVDA version:

2022.2.1, nvda_2022.3beta3, and the latest alphas.

#### Windows version:

Windows 10 21h2, spanish MX, 64 bits.

#### Name and version of other software in use when reproducing the issue:

VS Code, some windows consoles, some docker instances, Chrome and edge.

#### Other information about your system:
laptop brand: Dell.
CPU: i7 11800h.
Ram memory: 64gb ddr4 3200 mhz.
SSD on the operating system is installed: samsung 980 pro 2tb.
GPU: NVIDIA rtx 3070
Secoundary ssd where NVDA portable was executed: western digital nVMe 2tb, unable to determine the model.

#### Does the issue still occur after restarting your computer?
Yes. Not in the current installed version, but if you reinstall NVDA.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
nvda_2022.3beta2, the error does not exist.
NVDA 2022.2.1, 2022.3beta3, and the latest alphas, the error is present.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by seanbudd** [comment]
Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

**Comment by davidacm** [comment]
hi, this is the log using NVDA 2022.3beta3, I changed the language of NVDA to english to see the strings from the log in english.
[nvda-old.log](https://github.com/nvaccess/nvda/files/9420409/nvda-old.log)

**Comment by davidacm** [comment]
No problem, I restarted NVDA selecting the option "Restart with add-ons disabled". The issue is pressented, NVDA loaded the one core voices instead of my default synt. I can't remove the add-ons, but in a portable version without add-ons, the same issue is present.
[nvda-old.log](https://github.com/nvaccess/nvda/files/9420702/nvda-old.log)

**Comment by seanbudd** [comment]
@davidacm - does [this build](https://ci.appveyor.com/api/buildjobs/slyqnr8o058by9kp/artifacts/output%2Fnvda_snapshot_try-useDynamicWindowHandle-26315%2Cb38fb89c.exe) fix the issue?

**Comment by davidacm** [comment]
At least in the installer, the issue is still present, and the log is exactly the same:

ERROR - winAPI.sessionTracking.register (20:35:52.093) - MainThread (9348):
Unexpected error registering session tracking.
PermissionError: [WinError 5] Acceso denegado.

I should say again, that this issue disappears in the installed copy if I restart the computer, but appears again when I reinstall NVDA.

I will ask others if they have noticed the problem too, or is just this computer.

**Comment by seanbudd** [comment]
Thanks for the help with troubleshooting @davidacm, I think with this warning there is not much we can do. During the install you can always choose not to restart NVDA and allow the install to continue.
Devices generally need to be restarted after updating or installing NVDA.

**Comment by seanbudd** [comment]
Could you also show us a log of NVDA working correctly?

**Comment by seanbudd** [comment]
Closing as can't reproduce, the warning system is working as intended.

If anybody else can reproduce this, or the problem is recurring over a long time, please let us know so we can investigate further.
